ElvUI_MinimapButtons/ElvUI_MinimapButtons.lua

```diff
@@ -345,6 +346,10 @@ function addon:SkinMinimapButton(button)
 					if region:GetTexture() == "Interface\\Addons\\Outfitter\\Textures\\MinimapButton" then
 						region:SetTexture("Interface\\Icons\\INV_Chest_Cloth_43")
 					end
+				elseif name == "RecipeRadarMinimapButton" then
+					-- TODO: Only the "Normal" texture is set. The "Pushed" texture somehow remains as a blank texture (despite being
+					-- set to nil above), and shows up "blank" when clicking the icon. It's a minor issue but NO FIX has been found!
```

Comment:
I spent over 20 minutes trying all kinds of things, including not doing `SetPushedTexture(nil)` further up in the code, and investigating the OnClick handler of the button itself, but nothing worked. Finally I gave up, since it wasn't worth more of my time. It just means the icon flashes "blank" very briefly when clicked, since the "Pushed State" texture somehow remains on the object.